int32 n_usages        # e.g. 10
string source         # e.g. "text-json"
time creation_date    # (in UTC)
time last_access      # (in UTC)
        
# LTM software
string ltm_version    # e.g. "0.0.0"
string ros_version    # e.g. "kinetic"
string os_version     # uname -a information